server/channels/app/login_test.go

```diff
@@ -48,22 +48,31 @@ func TestCWSLogin(t *testing.T) {
 
 	t.Run("Should authenticate user when CWS login is enabled and tokens are equal", func(t *testing.T) {
 		token := model.NewToken(TokenTypeCWSAccess, "")
-		defer th.App.DeleteToken(token)
+		defer func() {
+			appErr := th.App.DeleteToken(token)
+			require.Nil(t, appErr)
+		}()
+
 		os.Setenv("CWS_CLOUD_TOKEN", token.Token)
 		user, err := th.App.AuthenticateUserForLogin(th.Context, "", th.BasicUser.Username, "", "", token.Token, false)
 		require.Nil(t, err)
```

Comment:
`err` is in fact a `*model.AppError`. Pleas update the variable name to `appErr`